native/src/components/__tests__/SearchListItem.spec.tsx

```diff
@@ -14,6 +14,7 @@ jest.mock('react-native-webview', () => ({
 describe('SearchListItem', () => {
```

Comment:
This file can be reverted, when you move the accessibilityHint inside the SearchListItem.tsx. See my comment in `SearchModal.tsx`

Reply:
@ztefanie I was unable to insert it into SearchListItem. It caused an undefined error